recipes/mimalloc/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "1.6.4":
```

Comment:
1.6.7 was released during this PR. Could we use this version please?

Reply:
Please remove the old version

Reply:
I agree. If someone needs the old version, they will add it explicitly.
And, for now, we will reduce the number of patches, which is great.

Reply:
Fixed in 1fd5adb3